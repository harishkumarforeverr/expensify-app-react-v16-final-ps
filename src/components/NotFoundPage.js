import React from 'react';
import {Link} from "react-router-dom";

const NotFoundPage=()=>{
    return (
        <div>
         <h1> NotFoundPage 404  </h1>
          <Link to="/">Go Home</Link>
        </div>
    )
}
export default NotFoundPage;