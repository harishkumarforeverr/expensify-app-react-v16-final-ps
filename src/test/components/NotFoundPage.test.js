import React from 'react';
import {shallow} from "enzyme";
import NotFoundPage from "../../components/NotFoundPage";

test('import NotFoundPage from "../../components/NotFoundPage"',()=>{
    const wrapper=shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})
