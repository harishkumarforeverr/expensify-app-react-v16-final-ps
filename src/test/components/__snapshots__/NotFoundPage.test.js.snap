// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import NotFoundPage from "../../components/NotFoundPage" 1`] = `
<div>
  <h1>
     NotFoundPage 404  
  </h1>
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
